packages/opds-web-client/src/hooks/__tests__/useDownloadButton-test.ts

```diff
@@ -105,19 +106,76 @@ describe("useDownloadButton", () => {
     const { result } = renderHook(
       () => useDownloadButton(undefined, "book-title"),
       {
-        wrapper: makeWrapper()
+        wrapper: makeWrapper().wrapper
       }
     );
     expect(result.current).to.equal(null);
   });
 
-  // it("fulfills direct links", () => {
-  //   const { result } = renderHook(
-  //     () => useDownloadButton(undefined, "book-title"),
-  //     {
-  //       wrapper: makeWrapper()
-  //     }
-  //   );
-  //   expect(result.current).to.equal(null);
-  // });
+  it("fulfills and downloads direct links", async () => {
+    const downloadStub = sinon.stub(download, "default");
+    const fulfillBookStub = sinon.stub();
+    const dispatchStub = sinon.stub();
+    const { wrapper, actions, store } = makeWrapper();
+    actions.fulfillBook = fulfillBookStub;
+    store.dispatch = dispatchStub;
+
+    const { result } = renderHook(
+      () => useDownloadButton(pdfMediaLink, "book-title"),
+      {
+        wrapper
+      }
+    );
+
+    // call fulfill
+    await result.current.fulfill();
```

Comment:
Is `current` part of the testing library API?

Reply:
yea, it stores the result of the hook in a ref, so the way to access it is `result.current`, and you can't destructure the result, since that creates a local copy instead of maintaining reference to the ref variable.